### PR TITLE
WIP - Retarget to 2021.2

### LINKED ISSUE #493: Update backend project to `netstandard2.0` or `net472`

### LINKED ISSUE #494: Update Gradle IntelliJ plugin
https://lp.jetbrains.com/gradle-intellij-plugin/

### PR DESCRIPTION
**Pull request by maartenba**
Do not merge yet, work in progress.

* Need to pin to proper EAP packages in `idea` 
* Can't compile - https://github.com/JetBrains/gradle-intellij-plugin/issues/674

Fixes #493
Fixes #494

### CODE REVIEW

File: BuildPlugin [hunk 1]
```diff
@@ -34,11 +34,11 @@ tc_open "Initializing build script"
     INJECT_INTELLIJ_VERSION=false
 
     IJ_VERSION_MAJOR_BUILD=211
-    IJ_VERSION_MAJOR=2021.1
+    IJ_VERSION_MAJOR=2021.2
     IJ_VERSION_MINOR=
     IJ_VERSION_LATEST=$IJ_VERSION_MAJOR.$IJ_VERSION_MINOR \
                       && [[ -z $IJ_VERSION_MINOR ]] && IJ_VERSION_LATEST=$IJ_VERSION_MAJOR
-    IJ_SCALA_VERSION_LATEST=2021.1.15
+    IJ_SCALA_VERSION_LATEST=2021.2.15
```
Comment (sdubov) [inline BuildPlugin]: I found the latest Scala plugin version is `2021.2.9`.

**Review by sdubov — APPROVED** [review]
Need to fix BuildPlugin script that cause failures in Plugin build and I think we can bump all related versions again )

**Comment by maartenba** [comment]
Still running into JetBrains/gradle-intellij-plugin#674 :-(

**Comment by maartenba** [comment]
Thanks @sdubov!

**Pull request merged**